### PR TITLE
[MB-5712] Dont exclude potential hhg ppm combo move in the move order fetcher

### PR DESCRIPTION
**Pull request by duncan-truss**
## Description

The TOO moves queue is supposed to exclude PPM moves but the change we made also excluded a combo HHG and PPM move when you create the PPM last in the customer flow.

We were relying on the value of the `selected_move_type` in the moves table to indicate when a move was an HHG or a PPM but this was inaccurate because it could end up with the value of PPM on a combo move.

We are already joining to the mto_shipments table, which should exclude moves that are only PPM but we have to remove the PPM selected_move_type from the exclusion list because it could be a combo move with an HHG. 

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make client_run
```

1. Create a new service member at milmovelocal:3000
2. Complete the user profile with an origin duty station in the 'LKNQ' GBLOC such as Yuma AFB
2. Next create an HHG shipment
3. Add another shipment of type PPM
4. Sign and submit your move
5. Login as the TOO user and the move should now appear in the queue

I've also changed the combo HHG+PPM move in the devseed test data to use the PPM SelectedMoveType so you can verify it appears in the TOO queue still.

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-5712) for this change

## Screenshots

Screen Shot 2020-11-25 at 10 24 02 AM

### CODE REVIEW

File: pkg/testdatagen/make_move.go [hunk 1]
```diff
@@ -186,6 +186,10 @@ func MakeHHGPPMMoveWithShipment(db *pop.Connection, assertions Assertions) model
 		Stub: assertions.Stub,
 	})
 
+	if !assertions.Stub {
+		mustSave(db, &move)
+	}
+
```
Comment (monfresh) [inline pkg/testdatagen/make_move.go:192]: Why is this necessary?

File: pkg/services/move_order/move_order_fetcher_test.go [hunk 2]
```diff
@@ -68,33 +68,40 @@ func (suite *MoveOrderServiceSuite) TestListMoves() {
...
-		suite.Equal(expectedMove.Orders.NewDutyStationID, move.Orders.NewDutyStation.ID)
-		suite.NotNil(move.Orders.Entitlement)
-		suite.Equal(*expectedMove.Orders.EntitlementID, move.Orders.Entitlement.ID)
-		suite.Equal(expectedMove.Orders.OriginDutyStation.ID, move.Orders.OriginDutyStation.ID)
-		suite.NotNil(move.Orders.OriginDutyStation)
-		suite.Equal(expectedMove.Orders.OriginDutyStation.AddressID, move.Orders.OriginDutyStation.AddressID)
-		suite.Equal(expectedMove.Orders.OriginDutyStation.Address.StreetAddress1, move.Orders.OriginDutyStation.Address.StreetAddress1)
+		suite.Equal(2, moveCount)
+
+		for i, move := range moves {
```
Comment (monfresh) [inline pkg/services/move_order/move_order_fetcher_test.go:88]: Are we confident that the order will always be the same? Also, I think that with the current pagination params, moves length will only be 1. I think we should remove pagination from our tests unless we're specifically testing it. It has caused confusion during test failures.

Comment (monfresh) [reply pkg/services/move_order/move_order_fetcher_test.go:88]: Maybe it would be better to have a separate test to make sure combo moves are included. That test would only test that the moveCount is correct since all we're trying to verify is that certain types of moves are returned. I'm not sure what value is provided by testing the model attributes unless we're specifically testing ordering, which I think we have a separate test for. If there are only 2 entries created in the DB, and 2 are returned, doesn't that imply that we got the correct entries?

File: pkg/testdatagen/make_move.go (hunk 1 above)
Comment (duncan-truss) [reply pkg/testdatagen/make_move.go:192]: 🤦 Yesterday was a long day

File: pkg/services/move_order/move_order_fetcher_test.go (hunk 2 above)
Comment (duncan-truss) [reply pkg/services/move_order/move_order_fetcher_test.go:88]: No I wasn't confident about the order and was likely going to specify an explicit sort order.  Happy to break it out into it's own test.

File: pkg/services/move_order/move_order_fetcher_test.go [hunk 3]
```diff
@@ -121,6 +124,21 @@ func (suite *MoveOrderServiceSuite) TestListMoves() {
...
+
+		suite.FatalNoError(err)
+		suite.Equal(2, moveCount)
+		suite.Len(moves, 2)
+
+		moveIDs := []uuid.UUID{moves[0].ID, moves[1].ID}
+
+		suite.Contains(moveIDs, expectedComboMove.ID)
+	})
+
```
Comment (monfresh) [inline pkg/services/move_order/move_order_fetcher_test.go:141]: Yay!

**Review by monfresh — APPROVED** [review]
LGTM
